control/tests/discrete_test.py

```diff
@@ -260,8 +258,6 @@ def testAddition(self, tsys):
             TransferFunction.__add__(tsys.siso_tf1c, tsys.siso_tf1d)
         with pytest.raises(ValueError):
             TransferFunction.__add__(tsys.siso_tf1d, tsys.siso_tf2d)
-        with pytest.raises(ValueError):
```

Comment:
I assume these are removed because these tests are already performed above in eg ‘test_timenase_conversion’?

Reply:
They don't raise an error anymore. You had an equivalent change in #431. `dt=0.1` and `dt=True` are compatible now.

Reply:
I see, makes sense!